plugins/hls-hlint-plugin/hls-hlint-plugin.cabal

```diff
@@ -2,11 +2,11 @@ cabal-version: 2.2
 name:          hls-hlint-plugin
 version:       0.1.0.0
 synopsis:      Hlint integration plugin with Haskell Language Server
-description:   Please see README.md
+description:   Please see Haskell Language Server Readme (https://github.com/haskell/haskell-language-server/blob/master/README.md)
 license:       Apache-2.0
 license-file:  LICENSE
 author:        Many,TBD when we release
-maintainer:    alan.zimm@gmail.com (for now)
+maintainer:    alan.zimm@gmail.com
```

Comment:
Shouldn't you be listed as the maintainer too @jneira ?

Reply:
We probably need some sort of generic reference to the hls team. Is that possible?

Reply:
I would prefer some collective thing too, but i am not against include myself if needed, of course.
I guess we would need a generic email account.

Reply:
Maybe just put the github repo there?

Reply:
mmm, that could work